i-slide.js

```diff
@@ -843,7 +843,22 @@ class ISlide extends HTMLElement {
   }
 }
 
-// Register the custom element
-customElements.define('i-slide', ISlide);
-
 export default ISlide;
+
+// We enable side-effects when hard-coded query string parameters are set
+if (import.meta && import.meta.url) {
+  const queryString = new URL(import.meta.url).search.slice(1);
+  const params = new URLSearchParams(queryString);
+  if (params.get("register") !== null || params.get("replaceSelector")) {
```

Comment:
Could `replaceSelector` be simplified as `selector`? I would argue that `replace` doesn't add more than characters here :)

Reply:
done in 8be8ae9